src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { User } from '../types';
+
+export type Props = {
+  user: User | null,
+};
```

Comment:
```suggestion
export type Props = {
  user: User,
};
```
We don't want to render UserInfo if we don't have a user. As in your solution we render empty link with invalid `href`